stable/kommander/requirements.lock

```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.3.3
+  version: 0.3.4
```

Comment:
v0.3.4 ?

Reply:
This is what `helm dependency update` did so I trust it, plus I've installed the chart and it works fine.

Reply:
All other dependencies are without the `v` prefix.

Reply:
Yeah, I stumbled upon that every time. 🤷‍♂